pkg/quarkus/v1alpha/api.go

```diff
@@ -116,3 +160,83 @@ func (p *createAPISubcommand) InjectResource(res *resource.Resource) error {
 
 	return nil
 }
+
+// findOldFilesForReplacement verifies marker (## marker) and if it found then merge new api CRD file to the odler logic
+func findOldFilesForReplacement(path, newfile string) bool {
+
+	f, err := os.Open(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// remember to close the file at the end of the program
+	defer f.Close()
+
+	// read the file line by line using scanner
+	scanner := bufio.NewScanner(f)
+	var foundMarker bool
+	for scanner.Scan() {
+		// do something with a line
+		if scanner.Text() == "## marker" {
+			foundMarker = true
+			break
+		}
+	}
+
+	if foundMarker {
+		scanner.Scan()
+		catLine := scanner.Text()
+
+		splitByPipe := strings.Split(catLine, "|")
+
+		finalString := strings.TrimSuffix(strings.TrimPrefix(strings.TrimSpace(splitByPipe[0]), "cat"), "target/kubernetes/kubernetes.yml")
+
+		updatedLine := "	" + "cat" + finalString + newfile + " target/kubernetes/kubernetes.yml" + " |" + splitByPipe[1]
+		// fmt.Printf("merge : %s\n", merge)
```

Comment:
Just another minor nit: We should remove this commented line.

Reply:
Ah! of course.